Project/src/MakeCall/IncomingCallCard.js

```diff
@@ -55,25 +55,37 @@ export default class IncomingCallCard extends React.Component {
                 <div className="ms-Grid-row text-center">
                     <span className="incoming-call-button"
                         title={'Answer call with microphone unmuted and video off'}
-                        onClick={async () => this.incomingCall.accept(await this.acceptCallMicrophoneUnmutedVideoOff())}>
+                        onClick={async () => {
+                            this.incomingCall.accept(await this.acceptCallMicrophoneUnmutedVideoOff());
+                            this.props.onReject();
```

Comment:
why are we calling onReject here?

Reply:
I took this part from another PR that had some initial scaffolding, I'm not sure why. It seems to work as intended. Will ask original author